src/Microsoft.DotNet.Build.Tasks.Feed/BuildManifest/WriteOrchestratedBuildManifestSummaryToFile.cs

```diff
@@ -34,8 +36,17 @@ public override bool Execute()
             string sdkProductVersion = model.Builds
                 .FirstOrDefault(b => b.Name == "cli")
                 ?.BuildId;
+
             string runtimeProductVersion = blobFeed.Artifacts.Blobs
-                .FirstOrDefault(b => b.Id.StartsWith("Runtime/"))
+                .FirstOrDefault(b =>
+                    b.Id.StartsWith("Runtime/") &&
+                    b.Id.EndsWith("/Microsoft.NET.CoreRuntime.2.1.appx"))
+                ?.Id.Split('/')[1];
+
+            string aspnetProductVersion = blobFeed.Artifacts.Blobs
+                .FirstOrDefault(b =>
+                    b.Id.StartsWith("Runtime/") &&
+                    b.Id.EndsWith("/aspnetcore_base_runtime.version"))
```

Comment:
Bringing forward this note: https://github.com/dotnet/buildtools/pull/1876/files/4672f7d739fb289367efa9a1380457f5a2af4568#diff-6ff3d24c7eabf53d4f76ac3464987901

> This logic that assumes a lot about the Core-Setup paths will be removed once we shuttle product version directly through the manifest. (See #1871.)

Reply:
I think this file should be fine. I have not plans to change this anytime soon.